Tasks/InstallAppleProvisioningProfile/installprovprofile.ts

```diff
@@ -0,0 +1,32 @@
+import path = require('path');
+import sign = require('ios-signing-common/ios-signing-common');
+import secureFilesCommon = require('securefiles-common/securefiles-common');
+import tl = require('vsts-task-lib/task');
+
+import { ToolRunner } from 'vsts-task-lib/toolrunner';
+
+async function run() {
+    let secureFileId: string;
+    let secureFileHelpers: secureFilesCommon.SecureFileHelpers;
+
+    try {
+        tl.setResourcePath(path.join(__dirname, 'task.json'));
+
+        if (tl.getInput('provisioningProfileLocation') === 'sourceRepository') {
+            let provProfilePath: string = tl.getInput('provProfileSourceRepository', true);
+
+            if (tl.filePathSupplied('provProfileSourceRepository') && tl.exist(provProfilePath)) {
```

Comment:
`tl.exist(provProfilePath)` emit warning (error?) if it doesn't exist?

(I'm noticing that we don't for the original case, so maybe not.)

Reply:
Good point, we should show an error here. In the original case it is not as important since we error if we are unable to download the file.